### PR TITLE
Simplify process to generate plotting documentation

### PR DESCRIPTION
**Pull request by jon-hagg**
### Purpose
The plots used on the docs website require specific scenarios as well as an environment to run the code snippets. The goal here is to simplify that as much as possible, so we can recreate the content as needed, e.g. to take advantage of new functionality.

### What the code is doing
What's involved:
* zip file with scenario data uploaded to blob storage
* docker-compose file for self contained environment
* notebook which generates the code used to run/save each code snippet

### Testing
I ran the notebook and compared the results with currently checked in files. There are a few minor differences, but that is due to inconsistencies in the code snippets, which we can fix separately if needed. 

### Usage Example/Visuals
If github will render the notebook, that is probably the most useful place to look

### Time estimate
20 min

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+Start the container, which will provide a notebook environment for running the code
+snippets.
+```
+docker compose up
+```
+
+After copying the notebook url from the output, start the `run_snippets.ipynb` notebook
+and run through it. For the most part, no interaction is required. The exception is any
+bokeh plots which must be manually saved as png (currently only the powerflow snapshot).
```
Comment (rouille) [inline docs/README.md:18]: I think we can use the following:
```
from bokeh.io import export_png

export_png(plot, filename="plot.png")
```
to automatically save the plot as png

**Comment by rouille** [comment]
Should we add a paragraph in **index.rst** that would explain how to download the zip file and spin up the container so users can run the code snippets? Plots would not be saved but would display only.

**Comment by rouille** [comment]
Now that we have **docs/run_snippets.ipynb**, we probably don't need **postreise/plot/demo/plot_tutorial.ipynb**. Correct?

**Comment by jon-hagg** [comment]
> Now that we have **docs/run_snippets.ipynb**, we probably don't need **postreise/plot/demo/plot_tutorial.ipynb**. Correct?

Might not hurt to have both.. the tutorial is more readable, while run_snippets is easier to run and save repeatedly. Maybe we could delete it down the road if it turns out we don't use it.

File: docs/README.md (hunk 1 above)
Comment (jon-hagg) [reply docs/README.md:18]: Couple of issues.. we can get the plot object to pass in using [curstate()](https://docs.bokeh.org/en/latest/docs/reference/io.html#bokeh.io.state.curstate) if `output_notebook()` was called too, but the export also required `pip install selenium`, and after that I got an error that firefox isn't supported. At that point I gave up.

**Review by rouille — APPROVED** [review]
Thanks

**Pull request merged**